sdk/tables/test-resources.json

```diff
@@ -37,13 +37,22 @@
             "virtualNetworkRules": [],
             "ipRules": [],
             "defaultAction": "Allow"
-        }
+        },
+      "customCosmosRoleName": "Azure Cosmos DB SDK role for Table Data Plane",
+      "customCosmosRoleDescription": "Azure Cosmos DB SDK role for Table Data Plane",
+      "customCosmosRoleActions": [
+        "Microsoft.DocumentDB/databaseAccounts/readMetadata",
+        "Microsoft.DocumentDB/databaseAccounts/tables/*",
+        "Microsoft.DocumentDB/databaseAccounts/tables/containers/*",
+        "Microsoft.DocumentDB/databaseAccounts/tables/containers/entities/*",
+        "Microsoft.DocumentDB/databaseAccounts/throughputSettings/read"
+      ]
     },
     "resources": [
         {
             "condition": "[equals(parameters('endpointType'),'storage')]",
             "type": "Microsoft.Authorization/roleAssignments",
-            "apiVersion": "[variables('authorizationApiVersion')]",
+            "apiVersion": "2018-09-01-preview",
```

Comment:
We should change this back to using the `authorizationApiVersion` variable and updating the value itself to whatever is the right version.

Reply:
@vcolin7 I can add a separate variable but from my testing I've found that storage and cosmos require different authorization api versions.